asapdiscovery/simulation/scripts/MProP0009_Sub.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+#BSUB -J run_single_simulation
+#BSUB -oo log_files/MProP0009_Sub.out
+#BSUB -e log_files/MProP0009_Sub.stderr
+#BSUB -n 1
+#BSUB -R rusage[mem=4]
+#BSUB -q gpuqueue
+#BSUB -W 12:00
+mkdir -p /data/chodera/asap-datasets/prepped_mpro_P0009/
```

Comment:
You should change this to make a directory in your own `/data/chodera` folder!

Reply:
Ah, I thought I made that distinction, will correct that!